### PR TITLE
Remove computeNode and use new analyzer interfaces for library retrieval

### PR DESCRIPTION
**Pull request by jcollins-g**
Refactors PackageGraph and Library construction to preserve the ResolvedLibraryResult objects from the analyzer, which contain the necessary AST nodes for Dartdoc ModelElements to be able to resolve source code and comment references.  This PR is the culmination of a joint effort with @bwilkerson, @scheglov and others, many thanks for all their efforts.

### CODE REVIEW

**Comment by jcollins-g** [comment]
Should be ready for review now with the publish of analyzer 0.33.5.

**Comment by jcollins-g** [comment]
Fixed a race condition in library detection found by the integration tests, but waiting for Flutter tests to pass on a republish of analyzer's dependencies that include dev-9.4 in their SDK ranges.

File: lib/src/model.dart [hunk 1]
```diff
@@ -6642,24 +6681,18 @@ class PackageBuilder {
     return metas;
   }
 
-  Future<List<LibraryElement>> _parseLibraries(Set<String> files) async {
-    Iterable<LibraryElement> libraries = new Iterable.empty();
+  Future<Map<LibraryElement, ResolvedLibraryResult>> _parseLibraries(Set<String> files) async {
+    Map<LibraryElement, ResolvedLibraryResult> libraries = new Map();
```
Comment (bwilkerson) [inline lib/src/model.dart]: Consider replacing "new Map()" with "{}".

**Comment by jcollins-g** [comment]
Another integration test failure for angular seems to be due to increased memory usage (since we're hanging on to many astNodes now this is not too surprising).  I'll see if there's something simple I can find in the profiler to make things better while waiting for the next publish.

**Pull request merged**